pkg/kritis/testutil/util.go

```diff
@@ -84,3 +85,12 @@ func getKey(key *openpgp.Entity, keyType string, t *testing.T) []byte {
 	wr.Close()
 	return gotWriter.Bytes()
 }
+
+func CreateSecret(t *testing.T, name string) *secrets.PgpSigningSecret {
```

Comment:
This helper seems trivial enough that it's not worth the additional cognitive overhead of needing to read a second file to understand the test.

Reply:
okies. i moved it to the file where we are using it. In case we need it else where, we can move it to test util.